qcfractal/alembic/versions/a2f76bb7be65_updates_collections_table_with_owner_.py

```diff
@@ -21,6 +21,7 @@ def upgrade():
     op.add_column('collection', sa.Column('description', sa.String(), nullable=True))
     op.add_column('collection', sa.Column('owner', sa.String(length=100), nullable=True))
     op.add_column('collection', sa.Column('view_url', sa.String(), nullable=True))
+    op.add_column('collection', sa.Column('view_url_plaintext', sa.String(), nullable=True))
```

Comment:
Could we come up with a name for this? It isn't quite plaintext in some ways. Nothing is coming to me at the moment, but I will think on it.

Also what if we add the `view_url` as `view_url_hdf5`? Or we can leave it as is and consider it *the* internal QCArchive internal format.

@loriab Any ideas on a name for XYZ + CSV version of shipping around datasets?

Reply:
I'm okay with changing `view_url` to `view_url_hdf5`.

The point of XYZ + CSV is that they are familiar, broadly compatible, and inspect-able with a text editor -- at the cost of performance. We could call it:
- `xyz+csv`
- `tar`
- `plain`

Reply:
`ascii`?

Reply:
`text`?

Reply:
Hmm, `plaintext` + `xyz+csv` is still my favorites.

Reply:
Still developer facing so some thought is needed. Happy to merge this as is if you wish.

Reply:
`TextTables`?

Reply:
I give in on plaintext, I don't think we will have a `plaintextv2`.